how-it-works/00_General/01_overview-of-the-internet-computer.subpage.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Architecture of the Internet Computer
+abstract: 
+shareImage: /img/how-it-works/overview-of-the-internet-computer.600.jpg
+slug: architecture-of-the-internet-computer
+---
+
+# Architecture of the Internet Computer
+
+The technology behind the Internet Computer (IC) has been reconceived from ground up, learning lessons from the weaknesses of earlier projects, e.g., the lack of scalability.
+The Internet Computer intends to become the World Computer, an open and secure blockchain-based network that can scale out to power humanity's computation.
+
+## Subnet Architecture
+
+The core part of the IC's architecture is its subnet architecture: The IC consists of many subnets, where each subnet is its own blockchain that operates concurrently with all the other subnets.
+New subnets can be added to scale the IC out, analogous to how public clouds scale out.
+Each subnet hosts smart contracts, called *canister smart contracts*, or simply *canisters*.
+Canisters on one subnet can send messages to canisters on the same or other subnet.
+The secure asynchronous cross-subnet (xnet) communication between canisters and the resulting loose coupling of subnets is one of the key principles that enable the scalability of the IC by means of adding new subnets.
+
+<figure>
+<img src="/img/how-it-works/subnet_architecture.png" alt="Architecture: The IC is composed of subnets, each of which is an independent blockchain" title="The IC is composed of subnets, each of which is an independent blockchain" align="center" style="width:600px">
+<figcaption align="center">
+The IC is composed of subnets, each of which is an independent blockchain.<br>
+Canisters within and across subnets communicate through asynchronous messaging.
+</figcaption>
+</figure>
+
+## Canister Smart Contracts
+
+Each subnet can host tens of thousands of *canister smart contracs* or simply *canisters*.
+A canister smart contract is a bundle comprising the smart contract Wasm bytecode and its storage.
```

Comment:
I think we should start by writing "web assembly (wasm)" and also link to documentation here.